core/domain/suggestion_jobs_one_off.py

```diff
@@ -369,3 +370,54 @@ def _update_community_contribution_stats_transactional(
 
         # Only yield the values from the transactions.
         yield (key_from_transaction, count_value_from_transaction)
+
+
+class PopulateFinalReviewerIdOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to populate final_reviewer_id property in the
+    suggestion models which does not have but expected to have one.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [suggestion_models.GeneralSuggestionModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        if not (item.status in [
+                suggestion_models.STATUS_ACCEPTED,
+                suggestion_models.STATUS_REJECTED] and (
+                    item.final_reviewer_id is None)):
+            return
```

Comment:
Do you still want to yield a count of untouched models to aid validation?

Reply:
It's fine to yield a count for every relevant branch in the code. This allows you to sanity-check that the sum of the counts matches the total number of models.